doc/ADVANCED.md

```diff
@@ -56,6 +56,23 @@ Following our [basic example](BASICS.md#a-basic-example), you will run gitarro a
 gitarro.rb -r openSUSE/gitarro -c "ruby-test" -d "ruby-gitarro-tuto" -g /tmp -t /tmp/tests.sh -f ".rb" -C
 ```
 
+# Caching Http in gitarro.
+
+Gitarro by default use faraday_http cache for storing and sparing rate limiting in github API.
```

Comment:
I would add a link from faraday_http cache to https://github.com/lostisland/faraday